### DISCUSSION TITLE
[Feature] Ability to upload images from anywhere on the phone

### DISCUSSION THREAD

**Discussion by 0verEngineer** [immich-app/immich#9152] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The ability to upload images from anywhere on the phone is a very crucial thing that is missing, when images are shared through messenger apps or something like that there is no possibility to upload them to immich.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7075
